client/src/components/Name.jsx

```diff
@@ -1,12 +1,12 @@
 class Name extends React.Component {
```

Comment:
is the Name component going to have state eventually?  If not, there is no need to make it a class, you can just use a functional component.